lib/ReactViews/Analytics/ParameterEditor.jsx

```diff
@@ -27,6 +27,177 @@ const ParameterEditor = React.createClass({
+                                />
+                            </div>);
+                }
+            }
+        },
+        {
+            id: 'rectangle',
+            parameterTypeToDiv: function ParameterTypeToDiv(type, parameterEditor) {
+                if (type === this.id) {
+                    return (<div>
+                                {parameterEditor.renderLabel()}
+                                <RectangleParameterEditor
+                                    previewed={parameterEditor.props.previewed}
+                                    viewState={parameterEditor.props.viewState}
+                                    parameter={parameterEditor.props.parameter}
+                                />
+                            </div>);
+                }
+            }
+        },
+        {
+            id: 'polygon',
+            parameterTypeToDiv: function ParameterTypeToDiv(type, parameterEditor) {
+                if (type === this.id) {
+                    return (<div>
+                                {parameterEditor.renderLabel()}
+                                <PolygonParameterEditor
+                                    previewed={parameterEditor.props.previewed}
+                                    viewState={parameterEditor.props.viewState}
+                                    parameter={parameterEditor.props.parameter}
+                                />
+                            </div>);
+                }
+            }
+        },
+        {
+            id: 'enumeration',
+            parameterTypeToDiv: function ParameterTypeToDiv(type, parameterEditor) {
+                if (type === this.id) {
+                    return (<div>
+                                {parameterEditor.renderLabel()}
+                                <EnumerationParameterEditor
+                                    previewed={parameterEditor.props.previewed}
+                                    viewState={parameterEditor.props.viewState}
+                                    parameter={parameterEditor.props.parameter}
+                                />
+                            </div>);
+                }
+            }
+        },
+        {
+            id: 'dateTime',
+            parameterTypeToDiv: function ParameterTypeToDiv(type, parameterEditor) {
+                if (type === this.id) {
+                    return (<div>
+                                {parameterEditor.renderLabel()}
+                                <DateTimeParameterEditor
+                                    previewed={parameterEditor.props.previewed}
+                                    parameter={parameterEditor.props.parameter}
+                                />
+                            </div>);
+                }
+            }
+        },
+        {
+            id: 'region',
+            parameterTypeToDiv: function ParameterTypeToDiv(type, parameterEditor) {
```

Comment:
The generic name `ParameterTypeToDiv` makes eslint happy, but it'd be better to use a more descriptive name, like `RegionParameterToDiv`.  That way the more useful name will show up in the debugger.